docs/getting_oss_rules_of_play_rating.md

```diff
@@ -0,0 +1,170 @@
+# Getting the OSS rules of play ratings
+
+The page describes how a rules of play rating may be calculated for an open source project.
+
+## Configuration
+
+Several of the rules need to configured as the detailed guidelines differ from project to project, for example which licenses may be allowed in your organization or which sections you want to enforce in the README file. Thus, in order to get proper OSS rules of play ratings, you need to provide some details. These details are usually passed to Fosstars via YAML configuration files for the respective data providers. Below you find the configurable data providers, information how to name the configuration files and the possible options in the respective configuration file. The section about the command-line tool further below explains how to pass these configuration files to Fosstars. The configuration that SAP uses for their OSS rules of play check run can be accessed in the [fosstars-oss-rules-of-play-sap-report](https://github.com/SAP/fosstars-rating-core/tree/fosstars-oss-rules-of-play-sap-report) branch.
```

Comment:
- Are there any mandatory iields in the configurations?
- Any special cases to be kept in check?

Reply:
Yep, for instance the expected license list is mandatory, others can also be omitted. But the details are explained in the respective sub-sections.